File: Python Exams/Basic Exam Collection/Basics Online Exam - 6 and 7 July 2019/2. Shoping/2. Shoping.py
budget = float(input())
card_numbers = int(input())
cpu_numbers = int(input())
ram_numbers = int(input())
card = card_numbers * 250
cpu =  cpu_numbers * (card * 0.35)
ram =  ram_numbers * (card * 0.10)

all_sum = card + cpu + ram

if card_numbers > cpu_numbers:
  all_sum -= (all_sum * 0.15)

total = abs(budget - all_sum)

if all_sum <= budget:
    print(f'You have {total:.2f} leva left!')
else:
 print(f'Not enough money! You need {total:.2f} leva more!')




'''
900
2
1
3
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 6 and 7 July 2019/2. Shoping/2. Shoping.js
function shoping(arg) {
 budget = arg[0]
 
 card = arg[1] * 250
 cpu  = arg[2] * (card * 0.35)
 ram  = arg[3] * (card * 0.10)

 sum = card + cpu + ram
 subtotal =  arg[1] > arg[2] ? sum-= (sum * 0.15) : sum
 total = Math.abs(budget - subtotal).toFixed(2)

 console.log( sum <= budget ? `You have ${total} leva left!` : `Not enough money! You need ${total} leva more!` );
}

//shoping(['900', '2', '1', '3'])

